src/components/DetailContainer/DetailContainer.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import DetailContainer from './DetailContainer';
+
+describe('DetailContainer', () => {
+  test('renders correctly', () => {
+    const { container } = render(<DetailContainer />);
+    expect(container.firstChild).toMatchSnapshot();
+  });
+  // TODO -> implement more tests, for instance if a tab has it's correct content
```

Comment:
Could you add with `test.todo('xxxxxx')` which scenarios would be missing? So another person can take over of it.
eg:
```
test.todo('should test click on tabs');
```